doc/operators.sgm

```diff
@@ -393,8 +393,10 @@
                 The center operator <literal>@@</literal> is a non-boolean
                 unary operator returning the center of an object.  In the
                 current implementation of <application>pgSphere</application>,
-                only centers of circles and ellipses are supported. Instead
-                of using the operator, you can use the function
+                only centers of circles, ellipses, points, array of points are
```

Comment:
Should be `arrays of points`.

Reply:
???
I also add center function and operator for single spoint for completeness interface

Reply:
Let me rephrase: Change "array of points" on this line to "arrays of points" to make this sentence grammatically correct.

Reply:
resolved